tests/test_raster.py

```diff
@@ -19,6 +19,7 @@
 from pylint.lint import Run
 from pylint.reporters.text import TextReporter
 
+import geoutils
```

Comment:
why don't use the import at the following line?

Reply:
good point :sweat_smile:, corrected!